core/data/tests/can_generate_anonymous_struct_with_skipped_fields/output.py

```diff
@@ -0,0 +1,38 @@
+"""
+ Generated by typeshare 1.11.0
+"""
+from __future__ import annotations
+
+from pydantic import BaseModel, Enum
+from typing import Union
+
+
+class AutofilledByUs(BaseModel):
+    """
+    Generated type representing the anonymous struct variant `Us` of the `AutofilledBy` Rust enum
+    """
+    uuid: str
+    """
+    The UUID for the fill
+    """
```

Comment:
I don't know if we test but according to the python styling guide, PEP8 we should uppercase acronyms if they are apart of the class and lowercase if they are a field. Maybe we want to add python to the uppercase acronym test?

Reply:
We can leave this as an improvement to be made in the external PR.